src/Features/Core/Portable/SimplifyInterpolation/Helpers.cs

```diff
@@ -73,7 +73,8 @@ private static IOperation Unwrap(IOperation expression)
         {
             if (expression is IInvocationOperation { TargetMethod: { Name: nameof(ToString) } } invocation &&
                 HasNonImplicitInstance(invocation) &&
-                !syntaxFacts.IsBaseExpression(invocation.Instance.Syntax))
+                !syntaxFacts.IsBaseExpression(invocation.Instance.Syntax) &&
+                !invocation.Instance.Type.IsRefLikeType)
```

Comment:
slightly worried that invocation.Instance might be null.  but i can't htink of a way to make that happen :)

Reply:
The HasNonImplicitInstance would seem to guarantee that there is an instance?

Reply:
great!